pilot/pkg/kube/inject/testdata/inject/auth.cert-dir.yaml.injected

```diff
@@ -13,6 +13,7 @@ spec:
       creationTimestamp: null
       labels:
         app: hello
+        istio_workload_mtls_ability: "true"
```

Comment:
Also the name is pretty bad - there is no special mtls ability here, and has nothing to do with the workload. 

The name should indicate the effect the label has: setting this label will enable MTLS traffic from clients to this workload.

How about:

istio_mtls: "true"

Reply:
is this for mTLS or is this to indicate as @mandarjog said, that the pod can do mTLS if we configure it? In other words, we simply need to indicate that the pod has Istio sidecar.. the rest is upto us.